data/templates/load-balancing/haproxy.cfg.j2

```diff
@@ -11,6 +9,14 @@ global
     daemon
 
 {% if global_parameters is vyos_defined %}
+{%     if global_parameters.logging is vyos_defined %}
+{%         for facility, facility_config in global_parameters.logging.facility.items() %}
+    log /dev/log {{ facility }} {{ facility_config.level }}
+{%         endfor %}
+{%     else %}
+    log /dev/log local0
```

Comment:
This keeps the current behavior:

> This does not provide flexibility to the user. Additionally, the current configuration logs all connection attempts across all services. This can be problematic for busy systems by producing large logs of unnecessary information.

I wonder if this should either be changed to something less verbose or be dropped at all to require the user to enable explicit logging.

Reply:
I can do that too. I would personally probably err on the side of requiring explicit configuration myself.

I was not sure what you all would prefer as far as maintaining how it functions prior to this change.

Reply:
@c-po updated, removed the default logging behavior in favor of explicit logging.